lib/msal-browser/FAQ.md

```diff
@@ -2,6 +2,12 @@
 
 # How do I get an authorization code from the library?
 
-# Iframe support
+Currently the msal-browser package is designed for Single-Page Applications that are handling all authentication through the browser client. We have not yet optimized this to work with server-side components. As such, requests to retrieve the authorization code from the first leg of the flow can't be met currently. We are currently working on an [implementation of msal that will run in node libraries](https://github.com/AzureAD/microsoft-authentication-library-for-js/projects/4), and as part of that we will explore options to make msal-browser work with server-side components.
 
-# B2C
+# Does this library work for iframed applications?
+
+We are currently working on support for iframed applications as well as solutions for applications affected by ITP 2.x changes. You can monitor the first of those tickets [here](#1410).
+
+# Will MSAL 2.x support B2C?
+
+We are currently working with the B2C service team to allow for authorization code flow to work in the browser with B2C tenants. We hope to have this available shortly after release.
```

Comment:
Don't we have the need for any other FAQs except work in progress tickets?

Reply:
We can add to it as we continue, I have only added questions I have received from the alpha.